ai-ml-samples/interactive/ImageClassificationInSpark.ipynb

```diff
@@ -0,0 +1,280 @@
+{
```

Comment:
<p>Change this to:</p><p> </p><h2>Environment Setup:</h2><ol><li>Create a Dataproc-enabled&nbsp;<a href="https://cloud.google.com/vertex-ai/docs/workbench/instances/create-dataproc-enabled" target="_blank">Vertex workbench</a>&nbsp;instance or use an existing instance.</li><li>Create a <a href="https://cloud.google.com/nat/docs/set-up-manage-network-address-translation#create-nat-gateway" target="_blank">Cloud NAT</a> gateway.</li><li>Create a <a href="https://cloud.google.com/dataproc-serverless/docs/quickstarts/jupyterlab-sessions#dataproc_create_serverless_runtime_template-JupyterLab" target="_blank">serverless runtime template</a>.</li></ol><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudDataproc/cloud-dataproc/pull/166/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Reply:
<p>I have kept some info about why NAT is needed to clarify and moved what notebook is doing into another section.</p>